src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/dates/ReleaseDateServiceTest.kt

```diff
@@ -457,7 +458,7 @@ class ReleaseDateServiceTest {
 
   @Nested
   inner class `Licence is due for early release` {
-    val date = LocalDate.of(2024, 4, 4)
+    val date = LocalDate.now().with(TemporalAdjusters.next(DayOfWeek.FRIDAY))
```

Comment:
would generally static import `TemporalAdjusters`

Reply:
(some general misgivings around moving to a mixture of some tests using dynamic now and others fixed now)